scripts/deploy-frontend.sh

```
@@ -0,0 +1,94 @@
+# Builds frontend with production environment variables and syncs to frontend_static volume
+
+set -e
+
+# Source utility functions
+SCRIPT_DIR="$(cd "$(dirname "${BASH_SOURCE[0]}")" && pwd)"
+source "${SCRIPT_DIR}/utils.sh"
+
+# Color codes
+GREEN='\033[0;32m'
+YELLOW='\033[1;33m'
+BLUE='\033[0;34m'
```

Shellcheck (0.11.0)

[warning] BLUE appears unused. Verify use (or export if used externally).

(SC2034)